core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -358,7 +358,7 @@ private void closeOrCancelCacheStream() {
         // server unreachable due to network partition, server busy due to alluxio worker is
         // busy, timeout due to congested network etc). But we want to proceed since we want
         // the user to continue reading when one Alluxio worker is having trouble.
-        LOG.warn("Cache stream close or cancel throws IOExecption {}, read continues.",
+        LOG.info("Cache stream close or cancel throws IOExecption {}, read continues.",
```

Comment:
change the message to:

`Closing / cancelling the cache stream encountered exception {}, reading from the regular stream will not be affected.`